### PR TITLE
Dutch localisation

### PR DESCRIPTION
**Pull request by edequartel**
### Summary
I have added the localisation file  from English into Dutch.

I work in the education department of [Bartimeus](www.bartimeus.nl). My role involves teaching ICT and developing software for learning Braille. Additionally, I coordinate all education related to Braille. Bartimeus is an organization dedicated to serving individuals with visual impairments from ages 0 to 80.

### Implementation details
For producing the localization file, I used ChatGPT. This will do for now.

### How to test it?
Change the language into Dutch
Simulator Screenshot - iPhone 13 - 2023-12-18 at 09 56 49

### CODE REVIEW

File: Accessibility Handbook.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1489,7 +1492,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Accessibility Handbook/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 278BFDM8KA;
```
Comment (giovaninppc) [inline Accessibility Handbook.xcodeproj/project.pbxproj:1495]: Hi @edequartel! Could you just remove your development team from the PR?

**Comment by giovaninppc** [comment]
Thanks for helping with the Accessibility Handbook 🚀

File: Accessibility Handbook.xcodeproj/project.pbxproj (hunk 1 above)
Comment (edequartel) [reply Accessibility Handbook.xcodeproj/project.pbxproj]: I have changed it and added the file to git ignore.

**Review by giovaninppc — APPROVED** [review]

**Comment by giovaninppc** [comment]
Thanks for your contribution @edequartel!
Changes will be available at AppStore later today

You can use our [Testflight beta link](https://testflight.apple.com/join/3eJsIxGn) for early access and to be notified of all new updates!

**Pull request merged**